app-backend/db/src/main/scala/com/azavea/rf/database/notification/templates/PlainGroupRequest.scala

```diff
@@ -0,0 +1,40 @@
+package com.azavea.rf.database.notification.templates
+
+import com.azavea.rf.database._
+import com.azavea.rf.datamodel._
+
+import doobie.ConnectionIO
+
+import java.util.UUID
+
+case class PlainGroupRequest(
+  groupId: UUID,
+  groupType: GroupType,
+  subjectId: String,
+  platformHost: String
+) {
+  def build: ConnectionIO[String] = {
+    for {
+      subjectEmail <- UserDao.unsafeGetUserById(subjectId).map((usr: User) => usr.email)
+      groupName <- groupType match {
+        case GroupType.Team =>
+          TeamDao.unsafeGetTeamById(groupId) map { (team: Team) => team.name }
+        case GroupType.Organization =>
+          OrganizationDao.unsafeGetOrganizationById(groupId) map {
+            (org: Organization) => org.name
+          }
+        case _ =>
+          throw new IllegalArgumentException(
+            "Users can only request to add themselves to organizations and teams"
+          )
+      }
+    } yield {
+      s"""
+      | User ${subjectEmail} has requested to join your ${groupType.toString.toLowerCase},
+      | ${groupName}.
+      | To approve or reject this request, visit the following page
+      | https://${platformHost}/jkjkjkjkjkjk
```

Comment:
@alkamin - can you help out here with the URL structure for the people management page?

Reply:
```
Subject: 
{Requester_Name} has requested to join  {Org_Name / Team_Name}

Body:
{Requester_Name} has requested your permission to join the {Org_Name / Team_Name} {Team / Organization}. To approve or reject this request, visit {Admin_Link}.

If you have questions, please feel free to reach out any time at *{Platform_Help_Email}*.

- The *{Platform_Name}* Team```